File: Arrays/2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.kt
class Solution {
    fun finalValueAfterOperations(operations: Array<String>): Int {
        var pos = 0
       
       for(num in operations){
           if( num == "++X" || num == "X++"){
               pos++
           }else if(num == "--X" || num == "X--"){
               pos--
           }
       }
       return pos
    }
}

File: TwoPointer/2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.swift
class Solution {
    func finalValueAfterOperations(_ operations: [String]) -> Int {
        var ans = 0

        for i in 0..<operations.count{
            if(operations[i] == "--X" || operations[i] == "X--"){
                ans -= 1
            }else if(operations[i] == "++X" || operations[i] == "X++"){
                ans += 1
            }
        }
        return ans
    }
}